Makefile

```diff
@@ -1,4 +1,13 @@
-.PHONY: lint
+.PHONY: lint build install uninstall
 
 lint:
 	shellcheck scripts/*
+
+build:
+	docker build -t open-balena-installer -f installer/Dockerfile .
```

Comment:
I think it shoul be fine to use “open-balena” instead of “open-balena-installer” throughout, this repo is the “installer” afterall.